api/egl.xml

```diff
@@ -1078,6 +1078,10 @@
             <unused start="0x3540" end="0x354F"/>
     </enums>
 
+    <enums namespace="EGL" start="0x3550" end="0x355F" vendor="QNX" comment="Reserved for Mike Gorchak of QNX (QNX platform and native buffer)">
+            <unused start="0x3550" end="0x355F"/>
+    </enums>
+
 <!-- Please remember that new enumerant allocations must be obtained by
      request to the Khronos API registrar (see comments at the top of this
      file) File requests in the Khronos Bugzilla, EGL project, Registry
```

Comment:
I should really edit this :-)

Reply:
I thought you wanted requesters to prepare the change. Sorry.

Reply:
I just meant that we haven't used Bugzilla for half a decade or more.